util/db/migrations/20210729104712-sfo-recreate-returns.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Returns', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+        type: Sequelize.INTEGER
+      },
+      ApplicationId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'Applications',
+          key: 'id'
+        }
+      },
+      beforeObjectiveRef: {
+        type: Sequelize.STRING
+      },
+      afterObjectiveRef: {
+        type: Sequelize.STRING
+      },
+      fromDate: {
+        type: Sequelize.DATE
+      },
+      toDate: {
+        type: Sequelize.DATE
+      },
+      comment: {
+        type: Sequelize.TEXT
+      },
```

Comment:
The same question, about possible 'nullable' fields, here too.

Reply:
resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/commits/d0befc3abf7f62d8acfa1d2233904ffdb86f9288